### PR TITLE
Point at users list rather than dev

### PR DESCRIPTION
**Pull request by brian-brazil**

### CODE REVIEW

File: resources/template.html [hunk 1]
```diff
@@ -219,7 +219,7 @@ <h4 class="panel-title">
     <footer>
       <p>
         Code: <a href="https://github.com/prometheus/pushgateway">prometheus/pushgateway</a>
-        Contact: <a href="mailto:prometheus-developers@googlegroups.com">prometheus-developers</a>
```
Comment (grobie) [inline resources/template.html]: What about just linking to our community page, so that we only have to care about one canonical page?

**Comment by brian-brazil** [comment]
Reworked it to make that work.

**Comment by grobie** [comment]
:+1:

**Comment by brian-brazil** [comment]
@beorn7 Do you have any comments on this?

IIRC this is here only so it doesn't look like the homepage an empty pushgateway isn't stuck loading, so the content doesn't matter much.

**Pull request merged**